samples/javascript/HttpStart/run.csx

```diff
@@ -0,0 +1,22 @@
+#r "Microsoft.Azure.WebJobs.Extensions.DurableTask"
+#r "Newtonsoft.Json"
+
+using System.Net;
+using System.Net.Http.Headers;
+
+public static async Task<HttpResponseMessage> Run(
```

Comment:
Hmm...it's really unfortunate that we have to have a c# function for this. It's actually possible to start orchestrations using JavaScript, as we demonstrate here: https://docs.microsoft.com/en-us/azure/azure-functions/durable-functions-instance-management#starting-instances.

```javascript
module.exports = function (context, input) {
    var id = generateSomeUniqueId();
    context.bindings.starter = [{
        FunctionName: "HelloWorld",
        Input: input,
        InstanceId: id
    }];

    context.done(null);
};
```
The problem I anticipate is that we don't have a way for JavaScript to generate the HTTP response that contains all the useful management links. An unfortunate oversight :(. Can you open an issue to track support for this?

I still think we should put the JavaScript starter function in the checked-in samples. It can live side-by-side with the C# one. For demos, we can use the C# version.

Reply:
I did create a JavaScript HttpStart function using that sample as a starting point. Unfortunately, the runtime failed at indexing because DurableOrchestrationClient isn't able to bind to string. We'd need to make that change in order for this to occur.